src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/model/Licence.kt

```diff
@@ -118,9 +112,6 @@ data class Licence(
   @Schema(description = "The email address for the supervising probation officer", example = "jane.jones@nps.gov.uk")
   val comEmail: String? = null,
 
```

Comment:
Did you intend to have `comEmail` on both licence and CommunityOffenderManager entities? Danger of getting out of step?

Reply:
This is not the entity, its the licence response DTO